### ISSUE TITLE
how to commit and push all the tabs

### ISSUE THREAD

**Issue by shouwei** [titoBouzout/SideBarGit#43]
Hi there,

I am using sieBarGit to manage my paper project. I already set a shortcut for Add&Commit&push command, it works pretty well for a single file or a single tab. 
Since I have to work on many files simultaneously, every time I want to commit, I have to switch to each tabs. Is there an efficient way to commit and push all the opening tabs at one time?

Thanks for your reading.

**Comment by titoBouzout** [comment]
no, but you can drag and drop the folder to the editor, right click the
folder, and select the same command

On Mon, Jul 11, 2016, 07:18 shouwei <notifications@github.com> wrote:

> Hi there,
>
> I am using sieBarGit to manage my paper project. I already set a shortcut
> for Add&Commit&push command, it works pretty well for a single file or a
> single tab.
> Since I have to work on many files simultaneously, every time I want to
> commit, I have to switch to each tabs. Is there an efficient way to commit
> and push all the opening tabs at one time?
>
> Thanks for your reading.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/titoBouzout/SideBarGit/issues/43>, or mute the thread
> <https://github.com/notifications/unsubscribe/AAD6nJQPbNDZ3Ef485_Z9A2C8eurbOnAks5qUhh2gaJpZM4JJOQO>
> .
>
-- 

__
Tito

**Comment by shouwei** [comment]
@titoBouzout 
Thanks for your reply.
That's would be an optional solution, but maybe not perfect for me. Because I am writing a paper with latex, and so many temporary files will be generated in the project folder during compilation. It's unnecessary to push these files.

May I suggest a new feature to SidebarGit, we say "all the opening tabs could be add&commit&push" in one time.
